Assets/MRTK/SDK/Features/UX/Scripts/Pointers/ParabolicTeleportPointer.cs

```diff
@@ -64,28 +64,38 @@ private void EnsureSetup()
 
         private static readonly ProfilerMarker OnPreSceneQueryPerfMarker = new ProfilerMarker("[MRTK] ParabolicTeleportPointer.OnPreSceneQuery");
 
+
+        private StabilizedRay stabilizedRay = new StabilizedRay(0.5f);
```

Comment:
Just to double check if you've checked that this change feels okay with controllers too

Reply:
the change feels great I promise!

Reply:
Awesome! Glad to see this change is bringing improvements to teleport across the board